src/components/forms/PlanForm/helpers.ts

```diff
@@ -773,7 +780,7 @@ export function getPlanFormValues(planObject: PlanDefinition): PlanFormFields {
       ? (taskGenerationStatusContext[0].valueCodableConcept as taskGenerationStatusType)
       : isDynamicPlan(planObject)
       ? taskGenerationStatuses.ignore
-      : taskGenerationStatuses.False;
+      : taskGenerationStatuses.internal;
```

Comment:
Does this affect all plan types?  If so, why is it necessary?

Reply:
This affects only the non dynamic plans. On creating non dynamic plans the task generation status should be set to internal

Reply:
@ciremusyoka have you confirmed that it is the case of Nigeria as well?